source/404.md

```diff
@@ -11,11 +11,11 @@ Use menu above to navigate or go to [home page](/).
 If you noticed any site problem, typos, etc on any page, 
 please correct using "Edit on GitHub" button or check on <https://github.com/apache/struts-site>.
```

Comment:
Thanks for your time ! IMHO maybe "you can", "you're able to" or "it's possible to" correct ... are better than "please correct" :)

Reply:
It is never enough to be polite. But that should be other PR